docs/source/release/v3.9.0/ui.rst

```diff
@@ -76,7 +76,6 @@ SliceViewer Improvements
 ------------------------
 
  - Added ability to view data using non orthogonal axes. View can be toggled on or off. When non orthogonal view is toggled, peak viewer and line vewer tools are disabled.
```

Comment:
Sorry I meant 78. This line here beginning "Added ability to view data using..."